enterprise/__init__.py

```diff
@@ -2,6 +2,6 @@
 Your project description goes here.
 """
 
-__version__ = "3.39.0"
+__version__ = "3.40.0"
```

Comment:
There is a database-related change but that doesn't cause compatibility issues, so minor version change is fine for me.